docs/ja/index.rst

```diff
@@ -286,7 +286,7 @@ Chronos は、出力した日時オブジェクトを表示するための多く
 
     Chronos::setTestNow(Chronos::now());
     MutableDateTime::setTestNow(MutableDateTime::now());
-    ChronosDate::setTestNow(ChronosDate::now());
+    ChronosDate::setTestNow(ChronosDate::parse(Chronos::now()));
```

Comment:
I don't love this API but its something we can simplify after this merges.

Reply:
This feels like some sort of search and replace error. It should just be now().